ResourceManagementOptionsConfiguration.cs

```diff
@@ -3,23 +3,25 @@
  * in any shape and let the resource manager know where these need to be included (head or foot).
  */
 
+using Microsoft.Extensions.Options;
 using OrchardCore.ResourceManagement;
 
 namespace Lombiq.TrainingDemo
 {
-    // ResourceManifest classes implement IResourceManifestProvider and possess the BuildManifests method. Don't forget
-    // to register this class with the service provider (see: Startup.cs). If you want to learn more about resources
-    // see: https://docs.orchardcore.net/en/dev/docs/reference/modules/Resources/
-    public class ResourceManifest : IResourceManifestProvider
+    // ResourceManagementOptionsConfiguration classes implement IResourceManifestProvider and possess the BuildManifests

```

Comment:
Már nem egészen az IResourceManifestProvider-t implementálja.

Reply:
Amúgy ennek az átállásnak mi is volt az értelme? Nem jöttem rá teljesen, hogy most mit kaptunk pluszban.

Reply:
Figyelni meg én fogok a meetingen.

Reply:
Köszi. Majd szólj, ha volt valami fontos.

Reply:
Mit képzeltek?!

Amúgy nem tudom, max annyi, hogy az Options erre is használható, így közelebb áll a standard ASP.NET Core-hoz.

Reply:
Jep, közelebb áll a standard ASP.NET Core-hoz, meg igazából az az interface ilyen szempontból felesleges volt.